app/experimenter/experiments/tests/test_models/test_nimbus_models.py

```diff
@@ -0,0 +1,71 @@
+from django.test import TestCase
+
+from experimenter.experiments.models import (
+    NimbusBucketNamespace,
+    NimbusExperiment,
+)
+from experimenter.experiments.tests.factories import (
+    NimbusBucketNamespaceFactory,
+    NimbusBucketRangeFactory,
+    NimbusExperimentFactory,
+)
+
+
+class TestNimbusExperimentModel(TestCase):
+    def test_str(self):
+        experiment = NimbusExperimentFactory.create(slug="experiment-slug")
+        self.assertEqual(str(experiment), experiment.name)
+
+
+class TestNimbusBucketNamespace(TestCase):
```

Comment:
Can you add a comment that describes which case this handles

e.g. An experiment is created that is orthogonal to all other experiments